tests/testthat/test-augsynth_plotting.R

```diff
@@ -0,0 +1,98 @@
+library( tidyverse )
+data(basque, package = "Synth")
+
+context("Test plotting features of single augsynth objects")
+
+basque <- basque %>% mutate(trt = case_when(year < 1975 ~ 0,
+                                            regionno != 17 ~ 0,
+                                            regionno == 17 ~ 1)) %>%
+    filter(regionno != 1)
+
+syn <- augsynth(gdpcap ~ trt, regionno, year, basque, progfunc = "Ridge", scm = T)
+
+test_that("All plot types for plot.summary.augsynth() are working", {
+
+    for (it in c('conformal', 'jackknife', 'jackknife+', 'permutation', 'permutation_rstat')){
+        s_syn <- summary(syn, inf_type = it)
+        for (pt in c('estimate only', 'estimate', 'placebo', 'outcomes', 'outcomes raw average')) {
+            if((!it %in% c('permutation', 'permutation_rstat')) & (pt == 'placebo')) {
+                expect_error(plot(s_syn, plot_type = pt))
+            } else {
+                p <- plot(s_syn, plot_type = pt)
+                expect_true('ggplot' %in% class(p))
+            }
+        }
+    }
+
+})
+
+test_that("All plot types for plot.augsynth() are working", {
+
+    for (it in c('conformal', 'jackknife', 'jackknife+', 'permutation', 'permutation_rstat')){
+        for (pt in c('estimate only', 'estimate', 'placebo', 'outcomes', 'outcomes raw average')) {
+            p <- plot(syn, plot_type = pt, inf_type = it)
+            expect_true('ggplot' %in% class(p))
+        }
+    }
+
+})
+
+
+test_that("General plotting functions do not crash", {
```

Comment:
Would be helpful to break this up into multiple tests with an informative title for each, rather than one large test testing multiple things.